### DISCUSSION TITLE
75 performance score of Next.js 14.1

### DISCUSSION THREAD

**Discussion by imran-baitham** [vercel/next.js#61865] [App Router] [answered]
## Can someone tell me why the performance score of Next.js 14.1's new release is only 75, and how can I improve it while using the app router?

![Screenshot from 2024-02-09 22-45-00](https://github.com/vercel/next.js/assets/81960197/2fe796a2-3139-4335-89d7-566ba9def1e6)

## I've tried a lot to improve the performance speed, but I was unsuccessful in doing so.

![Screenshot from 2024-02-09 22-51-51](https://github.com/vercel/next.js/assets/81960197/4d4d6a76-4ae5-4d4b-a6f8-3ac10c8a5dbd)

**Comment by DevDaniloFerrari** [comment, marked as answer]
Just try to build your application, run in production mode and do the performance analysis again.

To do this, run these commands:

`npm run build`
`npm start`

**here is my application running in development mode**
![image](https://github.com/vercel/next.js/assets/40414119/2fa41782-c531-4a95-a18b-edd9bafdcd33)

**here is my application running in production mode**
![image](https://github.com/vercel/next.js/assets/40414119/cf153463-8593-4518-b058-b41c2cc8e1d8)

**Reply by imran-baitham** [reply]
@DevDaniloFerrari Oh, I hadn't thought of that. Thank you for your help.

**Reply by DevDaniloFerrari** [reply]
You're welcome! :)

**Comment by Rashid-Aly** [comment]
@imran-baitham  Simple run these commands:
```
npm run build
npm start
```

**Comment by markoleavy** [comment]
It doesn't make much sense to me to use lighthouse with localhost, nor in dev or build/start mode.
